Annonce de recrutement d'eProfs L1

gtedavid opened the pull request:
Pull request
(c'est un brouillon nul, j'ai pas vraiment d'idée...)

gtedavid committed:
Create ANNOUCEMENT_ALL_LOOKING_FOR_EPROF_L1.md

TB reviewed by @Kadiatou-yatt @Grigaann

gtedavid committed:
Rename ANNOUCEMENT_ALL_LOOKING_FOR_EPROF_L1.md to ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md

@noftaly nevermind managed XD

noftaly commented on resources/ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md:
```diff
@@ -0,0 +1,7 @@
+Vous aimez bien (ré)expliquer un concept, aider d'autres étudiants de d'autres classes ?
+Mais est-ce que ça en vaut la peine ? Qu'est-ce que je peux avoir en retour ?
+
+C'est possible en devenant eProf.
+Vous avez la possibilité d'aider ceux qui posent des questions dans le chat, de faire des points en petits groupes ou sur le Discord, des sessions de révisions !
+
+ 
```
Je me permet de donner mon avis 😄 imo c'est très plat, pas très cohérent et pas très facile a comprendre... j'ai ajouté des emojis et reformulé un peu.

```suggestion
***NOUS RECRUTONS !***

Vous voulez __partager__ 🤝 ? Diffuser vos __connaissances__ 🗣 ? Aider vos pairs à __progresser__ 🎓 !?

Alors *devenez étudiant-prof* sur Ef'Réussite !
Vous pourrez répondre aux questions des étudiants et organiser des sessions de révisions pour des petits groupes ou pour une promotion entière ! Tout-cela en étant récompensé en retour ! *(PAVE, évènement de cohésion, et bien plus encore...)*
```

gtedavid replied:
Yes
J'avais fait l'annonce sur le tas sans trop  savoir exactement comment ça allait être formaté, se présenter 😅

Thx :)

grigan-tps replied:
- Je ne mettrais pas le point d'exclamation après "progresser"
- Je mettrais juste "étudiant-prof" en italique, pas le "devenez"
- Pas de tiret entre "tout" et "cela"
- Les points de suspension ne sont pas nécessaires, ils cassent le rythme de la phrase je trouve

noftaly committed:
Update resources/ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md

noftaly commented on resources/ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md:
```diff
@@ -0,0 +1,6 @@
+***NOUS RECRUTONS !***
+
+Vous voulez __partager__ 🤝 ? Diffuser vos __connaissances__ 🗣 ? Aider vos pairs à __progresser__ 🎓 !?
```
```suggestion
Vous voulez __partager__ 🤝 ? Diffuser vos __connaissances__ 🗣 ? Aider vos pairs à __progresser__ 🎓 ?
```

noftaly commented on resources/ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md:
```diff
@@ -0,0 +1,6 @@
+
+Vous voulez __partager__ 🤝 ? Diffuser vos __connaissances__ 🗣 ? Aider vos pairs à __progresser__ 🎓 !?
+
+Alors *devenez étudiant-prof* sur Ef'Réussite !
```
```suggestion
Alors devenez *étudiant-prof* sur Ef'Réussite !
```

noftaly commented on resources/ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md:
```diff
@@ -0,0 +1,6 @@
+Vous voulez __partager__ 🤝 ? Diffuser vos __connaissances__ 🗣 ? Aider vos pairs à __progresser__ 🎓 !?
+
+Alors *devenez étudiant-prof* sur Ef'Réussite !
+Vous pourrez répondre aux questions des étudiants et organiser des sessions de révisions pour des petits groupes ou pour une promotion entière ! Tout-cela en étant récompensé en retour ! *(PAVE, évènement de cohésion, et bien plus encore...)*
```
```suggestion
Vous pourrez répondre aux questions des étudiants et organiser des sessions de révisions pour des petits groupes ou pour une promotion entière ! Tout cela en étant récompensé en retour ! *(PAVE, évènement de cohésion, et bien plus encore)*
```

noftaly committed:
Apply suggestions from code review

gtedavid commented:
***NOUS RECRUTONS !***

Vous voulez __partager__ 🤝 ? Diffuser vos __connaissances__ 🗣 ? Aider vos pairs à __progresser__ 🎓 ?

Vous pourrez répondre aux questions des étudiants et organiser des sessions de révisions pour des petits groupes ou pour une promotion entière ! Tout cela en étant récompensé en retour ! *(PAVE, évènements de cohésion, et bien plus encore)*

gtedavid committed:
Update ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md

pluriel sur événements

gtedavid commented:
@melisandeGB

noftaly commented:
Il faudrait préciser quoi faire pour nous rejoindre.
Genre la si la personne se dit "ouais trop cool je veux venir", il/elle n'a aucune idée de quoi faire

gtedavid commented:
> Il faudrait préciser quoi faire pour nous rejoindre. Genre la si la personne se dit "ouais trop cool je veux venir", il/elle n'a aucune idée de quoi faire

ah oui euh

gtedavid commented:
> Il faudrait préciser quoi faire pour nous rejoindre. Genre la si la personne se dit "ouais trop cool je veux venir", il/elle n'a aucune idée de quoi faire

@noftaly 
Il y a le Form de recrutement que @iv-stpn avait fait mais comme il a caché le channel Horizon qui parlait de l'asso j'ai des doutes s'il vaut mieux pas juste en refaire un nouveau pour uniquement les eProfs ou juste pour tout Horizon

gtedavid committed:
Update ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md

**!!!! NOT READY TO BE PUBLISHED !!!**
Question about where to sign up

gtedavid commented:
**!!!! NOT READY TO BE PUBLISHED !!!**
Question about where to sign up needs to be added and answered

gtedavid committed:
Update ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md

!!! j'ai posté le lien vers le form pour relecture et modification
c'est pas prêt pour être publié

gtedavid commented:
!!! j'ai posté le lien vers le form pour relecture et modification @melisandeGB @Grigaann @Kadiatou-yatt 
c'est pas prêt pour être publié

gtedavid committed:
Update ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md

lien Form

noftaly commented on resources/ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md:
```diff
@@ -0,0 +1,8 @@
+Alors devenez *Étudiant-Prof* sur Ef'Réussite !
+Vous pourrez répondre aux questions des étudiants et organiser des sessions de révisions pour des petits groupes ou pour une promotion entière ! Tout cela en étant récompensé en retour ! *(PAVE, évènements de cohésion, et bien plus encore)*
+
+Remplis ce form :https://forms.gle/Lz4ywRukPP7uMWPZ6
```
```suggestion
Remplis ce formulaire : https://forms.gle/Lz4ywRukPP7uMWPZ6
```

gtedavid committed:
Update ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md

Update link

noftaly commented on resources/ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md:
```diff
@@ -0,0 +1,8 @@
+Alors devenez *Étudiant-Prof* sur Ef'Réussite !
+Vous pourrez répondre aux questions des étudiants et organiser des sessions de révisions pour des petits groupes ou pour une promotion entière ! Tout cela en étant récompensé en retour ! *(PAVE, évènements de cohésion, et bien plus encore)*
+
+Remplis ce form : https://bit.ly/EfRéussiteRecrute
```
```suggestion
Remplis ce formulaire : <https://bit.ly/EfRéussiteRecrute>
```

gtedavid committed:
Update resources/ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md

Co-authored-by: Elliot <elliot.maisl@gmail.com>

noftaly commented on resources/ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md:
```diff
@@ -0,0 +1,8 @@
+Alors devenez *Étudiant-Prof* sur Ef'Réussite !
+Vous pourrez répondre aux questions des étudiants et organiser des sessions de révisions pour des petits groupes ou pour une promotion entière ! Tout cela en étant récompensé en retour ! *(PAVE, évènements de cohésion, et bien plus encore)*
+
+Remplis ce formulaire : <https://bit.ly/EfRéussiteRecrute>
```
```suggestion
[Remplis ce formulaire](https://forms.gle/Lz4ywRukPP7uMWPZ6)
```

gtedavid commented:
New Pull Request made 

Announcement was posted
https://github.com/horizon-efrei/efreussite-webhooks/pull/9

noftaly committed:
Rename ANNOUNCEMENT_ALL_LOOKING_FOR_EPROF_L1.md to ANNOUNCEMENT_ALL_RECRUTEMENT_EPROF_L1_MARS22.md
